NetKet/Graph/custom_graph.hpp

```diff
@@ -47,34 +44,13 @@ class CustomGraph : public AbstractGraph {
 
   std::vector<std::vector<int>> automorphisms_;
 
-  bool isbipartite_;
```

Comment:
Why? This info is used for optimisation.

Reply:
Then you need to store also adjacency matrices etc, if you don't want to compute them every time you call the corresponding method. Ideally you need a flag that stores the state of the computation and possibly avoids recomputing everything from scratch if already computed.

Reply:
It's not about recomputing. `CustomGraph` now always assumes that it's not bipartite. I suggest we leave the argument there for now and add implementing two-coloring to the TODO list. [this](https://github.com/netket/netket/blob/9570034e38d98846229015bb3c5676e84cb5ff61/NetKet/Hamiltonian/heisenberg.hpp#L59) has a *huge* impact on convergence.

Reply:
There is already an open issue about that #48 , that's why I just set up a warning. We can integrate this at a later stage. And by the way, the impact on the convergence is only if you don't do things properly :)

Reply:
> There is already an open issue about that #48

I see.

> And by the way, the impact on the convergence is only if you don't do things properly :)

Out of curiosity, what do you mean by "not doing thing properly"?